R-package/R/lgb.Booster.R

```diff
@@ -512,6 +520,21 @@ Booster <- R6::R6Class(
         start_iteration <- 0L
       }
 
+      # Process data frame if required
+      if (is.data.frame(data)) {
+        if (private$is_from_data_frame) {
+          data <- Dataset$public_methods$process_data_frame_columns(
+                    data,
+                    private$colnames,
+                    private$categorical_feature,
+                    private$factor_levels
+                  )
```

Comment:
```suggestion
          data <- Dataset$public_methods$process_data_frame_columns(
              data = data
              , colnames = private$colnames
              , categorical_feature = private$categorical_feature
              , factor_levels = private$factor_levels
          )
```

Please match the style in used in the rest of the R package, which is comma-first and closing parentheses vertically aligned with the beginning of the line.

And please use keyword arguments for any function calls to internal methods with more than one argument. This makes the code a bit easier to read and prevents bugs caused by mistakes in the order that arguments are given.